src/Commands/AutocompleteCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Pantheon\Terminus\Commands;
+
+use Pantheon\Terminus\Exceptions\TerminusNotFoundException;
+use Pantheon\Terminus\Helpers\LocalMachineHelper;
+
+/**
+ * Class AutocompleteCommand
+ * @package Pantheon\Terminus\Commands
+ */
+class AutocompleteCommand extends TerminusCommand
+{
+    
+    /**
+     * Displays Terminus Autocomplete config
+     *
+     * @command autocomplete
+     *
+     * @usage Returns the shell autocomplete config.
+     */
+    public function autocomplete()
+    {
+        return $this->retrieveList();
+    }
+
+    /**
+     * Return the filename.
+     *
+     * @return string
+     */
+    protected function getFilename()
+    {
+        return $this->config->get('assets_dir') . "/autocomplete.txt";
```

Comment:
This should go into the cache dir. The assets dir holds the ASCII art.

Reply:
I'll adjust the location

Reply:
@TeslaDethray so my intent with this was to cache the autocomplete script so it could be included with the Phar instead of including all the libraries used to generate it. The cache dir looks like it's user specific, and wouldn't be bundled into the phar. Would it make sense to create a separate folder just for caching the autocomplete script?